Sources/Config.php

```diff
@@ -745,7 +745,7 @@ class Config
 				 * The absolute path to the forum's folder. (not just '.'!)
 				 */
 				END,
-			'default' => 'dirname(__FILE__)',
+			'default' => '__DIR__',
```

Comment:
This is acceptable, but only if the same change is made in ./other/Settings.php.